packages/flutter_tools/lib/src/tester/flutter_tester.dart

```diff
@@ -132,16 +132,13 @@ class FlutterTesterDevice extends Device {
     await bundle.build(
       mainPath: mainPath,
       assetDirPath: assetDirPath,
-      applicationKernelFilePath: applicationKernelFilePath,
+      outputFilePathNoExtension: applicationKernelFilePath,
       precompiledSnapshot: false,
       trackWidgetCreation: buildInfo.trackWidgetCreation,
     );
     command.add('--flutter-assets-dir=$assetDirPath');
 
-    // TODO(scheglov): Either remove the check, or make it fail earlier.
```

Comment:
I don't think this can be null. I am guessing this is just a TODO that was never removed. I can strip this line from the CL as it isn't really related. @scheglov

Reply:
No objections.